fix(api): Add set_calibration not implemented to versioning doc (#12274)
OpenTrons/opentrons-api: api/docs/v2/versioning.rst

  - The ``height`` parameter of :py:meth:`.MagneticModuleContext.engage` was removed.
    Use ``offset`` or ``height_from_base`` instead.
    

>   - ``Labware.separate_calibration`` was removed,
>     since it was a holdover from a calibration system that no longer exists.
>     :py:meth:`.Labware.set_calibration` was removed.
>     Setting a labware's calibration after it's been loaded is not supported.

Author: ```suggestion
  - ``Labware.separate_calibration`` and :py:meth:`.Labware.set_calibration` were removed,
    since they were holdovers from a calibration system that no longer exists.
```

>   - ``Labware.separate_calibration`` and ``ModuleContext.separate_calibration`` were removed,
>     since they were holdovers from a calibration system that no longer exists.
>     :py:meth:`.Labware.set_calibration` was removed.
>     Setting a labware's calibration after it's been loaded is not supported.

Reviewer: So I was thinking more on the lines of-
```suggestion
  - ``Labware.separate_calibration``, :py:meth:`.Labware.set_calibration` and ``ModuleContext.separate_calibration`` were removed, since they were holdovers from a calibration system that no longer exists.
```

    because it made unsafe modifications to the protocol's geometry system,
    and the Thermocycler's "semi" configuration is not officially supported.

  - :py:meth:`.Labware.set_calibration` is not supported on this API version.
    Setting a labware's calibration after it's been loaded is not supported.

> 

Reviewer: I think we can just mention this method on line 299 above which already talks about removed calibration methods in 2.14

  - :py:meth:`.Labware.set_offset` is not yet supported on this API version.
    Run protocols via the Opentrons App, instead.

>   - :py:meth:`.Labware.set_calibration` is not supported on this API version.

Reviewer 2: is `set_calibration` ever going to be supported again? should it go in the removals section, even if it's thematically linked to `set_offset`?

Author: you are right @ecormany we should probably move it to the removals section. I will do that change
